debug_module.tex

```diff
@@ -217,6 +217,31 @@ \section{Run Control} \label{runcontrol}
 until cleared by the debugger writing 1 to \Fclrresethaltreq
 while the hart is selected, or by DM reset.
 
+\section{Halt Groups and External Triggers}
+
+An optional feature allows a debugger to create halt groups. When any hart in a
```

Comment:
elsewhere in this document we ahve things like "lists of features supported by this specification". If this list still exists, should this be added to that list?